commons/pdf-commons/src/main/java/se/idsec/signservice/security/sign/pdf/configuration/PDFAlgorithmRegistry.java

```diff
@@ -302,13 +302,39 @@ public static String getAlgorithmURI(final ASN1ObjectIdentifier sigAlgoOid, fina
     return supportedAlgoMap.keySet()
       .stream()
       .map(s -> supportedAlgoMap.get(s))
-      .filter(algoProp -> algoProp.getSigAlgoOID().equals(sigAlgoOid) &&
+      .filter(algoProp -> isSigAlgoEquivalent(algoProp.getSigAlgoOID(), sigAlgoOid, digestAlgoOid) &&
           algoProp.getDigestAlgoOID().equals(digestAlgoOid))
       .map(PDFSignatureAlgorithmProperties::getSigAlgoId)
       .findFirst()
       .orElseThrow(() -> new NoSuchAlgorithmException("Non supported combination of signature algorithm and hash algorithm"));
   }
 
+  /**
+   * This function is designed to allow identifiers for RSA encryption to be equivalent to identifiers for various RSA combined with various hash functions
+   * @param sigAlgoPropOID
```

Comment:
Missing documentation for param and return

Reply:
Fixed